src/main/java/com/ebivariation/contigalias/repo/AssemblyRepository.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2020 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ebivariation.contigalias.repo;
+
+import com.ebivariation.contigalias.entities.AssemblyEntity;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.Optional;
+
+@Repository
+public interface AssemblyRepository extends JpaRepository<AssemblyEntity, Long> {
+
+    default Optional<AssemblyEntity> findAssemblyEntityByAccession(String accession) {
+        return this.findAssemblyEntityByGenbankOrRefseq(accession, accession);
+    }
+
+    Optional<AssemblyEntity> findAssemblyEntityByGenbankOrRefseq(String genbank, String refseq);
+
+    long countByIdNotNull();
```

Comment:
```suggestion
    long count();
```
I tried this and it works.